openmc/geometry.py

```diff
@@ -482,22 +482,30 @@ def get_lattices_by_name(self, name, case_sensitive=False, matching=False):
         lattices.sort(key=lambda x: x.id)
         return lattices
 
-    def determine_paths(self):
+    def determine_paths(self, instances_only=False):
         """Determine paths through CSG tree for cells and materials.
 
         This method recursively traverses the CSG tree to determine each unique
         path that reaches every cell and material. The paths are stored in the
         :attr:`Cell.paths` and :attr:`Material.paths` attributes.
 
+        Parameters
+        ----------
+        instances_only : bool
```

Comment:
I know this isn't convention in OpenMC, but have you considered noting what arguments are optional like in Numpy?